.github/workflows/ci.yml

```diff
@@ -32,6 +32,12 @@ jobs:
       - uses: actions/checkout@v4
       - name: Bootc Ubuntu Setup
         uses: ./.github/actions/bootc-ubuntu-setup
+      - name: Setup Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-all-crates: true
+          save-if: ${{ github.ref == 'refs/heads/main' }}
+          lookup-only: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
Why are you setting `lookup-only`?

Reply:
This will ensure that on `main`, no cache will be used and everything will be built from scratch.
But if you don't like it, it doesn't have to be there.